#! /usr/bin/env python

PACKAGE='jsk_perception'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       name    type     level     description     default      min      max
gen.add("score_thresh", double_t, 0, "Threshold for confidence score", 0.70, 0.0, 1.0)

exit(gen.generate(PACKAGE, PACKAGE, "MaskRCNNInstanceSegmentation"))
